Dockerfile

```diff
@@ -10,24 +10,30 @@ RUN mvn clean install -DskipTests
 
 FROM build-hapi AS build-distroless
 RUN mvn package spring-boot:repackage -Pboot
-RUN mkdir /app && \
-    cp /tmp/hapi-fhir-jpaserver-starter/target/ROOT.war /app/main.war
+RUN mkdir /app && cp /tmp/hapi-fhir-jpaserver-starter/target/ROOT.war /app/main.war
 
-FROM gcr.io/distroless/java-debian11:11 AS release-distroless
+
+########### bitnami tomcat version is suitable for debugging and comes with a shell
+########### it can be built using eg. `docker build --target tomcat .`
+FROM bitnami/tomcat:9.0 as tomcat
```

Comment:
When building and starting the image as-is, I get a 

```
org.h2.jdbc.JdbcSQLNonTransientException: Error while creating file "/target" [90062-210]
...
Caused by: java.nio.file.AccessDeniedException: /target
```

we might have to create a `/target` folder first and set appropriate permissions.